### ISSUE TITLE
consider not setting EPP to balance_power when on battery

### ISSUE THREAD

**Issue by stereomato** [intel/intel-lpmd#68]
https://github.com/intel/intel-lpmd/commit/6473934c5c46cd915251ca2fda6ac1e0ae3bf156#diff-fb997c21d1ef39b08732f059fb512ee26cd72af55a8339061e8054a27288c856R610 does this.

See https://gitlab.freedesktop.org/upower/power-profiles-daemon/-/issues/151 for why this is a bad idea. Also, wasn't Intel's whole deal the "race to idle" thing?

**Comment by zhang-rui** [comment]
Hi @stereomato 
thanks for raising this, give me some time to grab more details before making any changes.

**Comment by spandruvada** [comment]
It is not that we want to set different EPP unless user defines a config entry, but we don't want to override what PPD set. I think PPD is still setting balance_power EPP on DC. If LPMD service is stopped then it needs to restore the default EPP, which PPD would have set as it may race with that.

**Comment by stereomato** [comment]
tuned-ppd (which is default on fedora, included on arch repos and soon to be on nixOS which i'm currently running) lets you configure what epp is set when changing power source. I set it to balance-performance because for intel platforms is what supposedly intel said was the most appropriate + balance_power leads to laggy UI across desktops (happens on gnome and kde, more so on kde)

**Comment by stereomato** [comment]
Yeah, actually, this was due to a change in power-profiles-daemon that set epp to balance_power, sorry!

**Comment by stereomato** [comment]
hmmm, I have ppd patched to not do that, but i still see lpmd setting stuff to balance_power

![Image](https://github.com/user-attachments/assets/2c4a0da5-ee1a-418b-aedb-c3d814a79822)
